rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -1995,8 +1996,40 @@ message ClaimSwapNeuronsRequest {
     repeated NeuronId followees = 8;
   }
 
+  message NeuronsFund {
+    optional uint64 nns_neuron_id = 1;
+    optional ic_base_types.pb.v1.PrincipalId nns_neuron_controller = 2;
+    optional ic_nervous_system.pb.v1.Principals nns_neuron_hotkeys = 3;
+  }
+
+  message Direct {}
+
+  message NeuronRecipe {
```

Comment:
add docs for the new types and parameters

Reply:
Done